src/tests/verify_resources.py

```diff
@@ -436,6 +438,23 @@ def test_14_res_contacts_match(rgs, rgfns, contacts):
 
     return errors
 
+def test_16_Xrootd_DNs(rgs, rgfns):
+    # verify each Xrootd service has DN
+
+    errors = 0
+
+    for rg, rgfn in zip(rgs, rgfns):
+        if rgfn.endswith('FACILITY.yaml'):
```

Comment:
I don't think any of the resource group file names (in `rgfns`) are the `FACILITY` files, since those are one level higher, so this `if`/`continue` block looks unnecessary.  (Unless i'm missing something?)